anago

```diff
@@ -1752,13 +1752,11 @@ if ((FLAGS_stage)); then
   logecho
   logecho "To release this staged build, run:"
   logecho
-  logecho "$ gcbmgr release ${EXTRA_FLAGS[*]} $RELEASE_BRANCH" \
-          "--buildversion=$JENKINS_BUILD_VERSION"
+  logecho "$ gcbmgr release ${EXTRA_FLAGS[*]} $RELEASE_BRANCH --buildversion=$JENKINS_BUILD_VERSION"
```

Comment:
```suggestion
  logecho -n "$ gcbmgr release ${EXTRA_FLAGS[*]} $RELEASE_BRANCH --buildversion=${JENKINS_BUILD_VERSION}\n"
```

I think `logecho -n` is what you are after:
`-n` does not use `fmt` to format the text, but still keeps the `$log_prefix` (which `-r` would set to `""`), see: 

https://github.com/kubernetes/release/blob/2b743b69e411ee2acc359f300a50b0d94ee7ee83/lib/common.sh#L127-L134

https://github.com/kubernetes/release/blob/2b743b69e411ee2acc359f300a50b0d94ee7ee83/lib/common.sh#L142-L148

Reply:
The -n in theory only controls the trailing newline, but...

Looking closer at the implementation, I think what's happened for me is that $fmtlen is shorter than the long lines when a --buildversion is display, so fmt is inserting a newline.  My patch wont change that.

Adding a -n though would mean fmt is not used, meaning the a long line is not split which is my goal, although I'd need to follow with a plain logecho to get a new line after the intended output.